### PR TITLE
build: run offline_compiler_test using bazel

### PR DESCRIPTION
**Pull request by devversion**
Switches the `offline_compiler_test` to Bazel. The old script was intentionally disabled and also sort of outdated now (some things had to be adjusted).

The whole integration test runs in a real integration-test fashion, just under Bazel without utilizing any rules such as `ng_module` or `ts_library`. This ensures that we still test non-Bazel code while slowly moving the Angular source repo to Bazel.

Also Windows compatible 😄

### CODE REVIEW

File: packages/compiler-cli/integrationtest/test/i18n_spec.ts [hunk 1]
```diff
@@ -36,8 +36,8 @@ const EXPECTED_XMB = `<?xml version="1.0" encoding="UTF-8" ?>
 <messagebundle>
   <msg id="126808141597411718"><source>node_modules/third_party/other_comp.d.ts:1,2</source>other-3rdP-component
 multi-lines</msg>
-  <msg id="8136548302122759730" desc="desc" meaning="meaning"><source>src/basic.ts:1</source>translate me</msg>
-  <msg id="3492007542396725315"><source>src/basic.ts:5</source><source>src/entry_components.ts:1</source>Welcome</msg>
```
Comment (alexeagle) [inline packages/compiler-cli/integrationtest/test/i18n_spec.ts]: I don't see how this change is related to re-hosting the tests. If you've added some unrelated fixes/refactorings, please put them in either a different PR or separate commits on this one.
Saw this other places like packages/compiler-cli/integrationtest/src/module.ts
The more focused the PR can be the easier it is to review.

Comment (devversion) [reply packages/compiler-cli/integrationtest/test/i18n_spec.ts]: Yeah these are actually fixed in order to get the outdated tests pass. We need to fix these because enabling the test with failures would be undesired. 

Agreed that it would make sense to add these to separate commits. I've just done this! 

<sup>FYI: The small removal of Angular Material remains under the conversion commit since I consider it as part of the conversion that is intentionally left out due to it being out-of-scope for this integration test. It's too complicated to install Material in a non-cyclic way & also with Bazel)</sup>

**Review by IgorMinar — APPROVED** [review]
thanks doing this! awesome work

**Comment by IgorMinar** [comment]
merge-assistance: invoking global approval, and please ignore pullapprove.
